src/auto/cell_layout.rs

```diff
@@ -94,9 +97,31 @@ impl<O: IsA<CellLayout>> CellLayoutExt for O {
     //    unsafe { TODO: call ffi::gtk_cell_layout_set_attributes() }
     //}
 
-    //fn set_cell_data_func<'a, P: IsA<CellRenderer>, Q: Into<Option<&'a /*Unimplemented*/CellLayoutDataFunc>>, R: Into<Option</*Unimplemented*/Fundamental: Pointer>>>(&self, cell: &P, func: Q, func_data: R, destroy: /*Unknown conversion*//*Unimplemented*/DestroyNotify) {
-    //    unsafe { TODO: call ffi::gtk_cell_layout_set_cell_data_func() }
-    //}
+    fn set_cell_data_func<P: IsA<CellRenderer>, Q: Fn(&CellLayout, &CellRenderer, &TreeModel, &TreeIter) + Send + Sync + 'static, R: Into<Option<Q>>>(&self, cell: &P, func: R) {
```

Comment:
Send+Sync?

Reply:
Yes that's wrong. Where does that come from? :)